src/LanguageServer/Test/HoverTests.cs

```diff
@@ -224,6 +224,18 @@ class Box(Generic[_T]):
             AssertHover(hs, analysis, new SourceLocation(6, 8), @"some: str", new SourceSpan(6, 8, 6, 12));
         }
 
+        [TestMethod, Priority(0)]
+        public async Task AssignmentExpressions() {
+            const string code = @"
+a = 5
```

Comment:
You probably don't want this extra assignment; if the analysis were to skip over the `:=` then the test may pass because the normal assignment caused `a` to be an `int`.